src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -1157,6 +1158,47 @@ static string GenerateStorageParameterValue(object value)
         bool VersionAtLeast(int major, int minor)
             => _postgresVersion is null || new Version(major, minor) <= _postgresVersion;
 
+        string IndexColumnList(string[] columns, string[] operators)
```

Comment:
As above, any reason not to use the interfaces?

Reply:
It's an internal helper method, I didn't think much about it.

Reply:
As above, please disregard.